src/commands/generateRunbook.ts

```diff
@@ -50,15 +50,24 @@ const prepareRunbookFile = async (workspaceFolder: vscode.WorkspaceFolder, promp
 export const generateRunbook = (secrets: vscode.SecretStorage) => vscode.window.withProgress({ location: vscode.ProgressLocation.Window }, async progress => {
     // Check docker command exists
     try {
-        const res = spawnSync("docker", ["--version"]);
+        const res = spawnSync("docker", ["version"]);
+
         if (res.error) {
             throw res.error;
         }
-    } catch (e) {
-        return vscode.window.showErrorMessage("Docker not found", { modal: true }, "Install Docker Desktop").then((value) => {
+
+        if (res.status !== 0) {
+            throw new Error(`Docker command exited with code ${res.status} and output the following error: ${res.error || res.stderr.toString()}`);
+        }
+
+    } catch (e: unknown) {
+        return vscode.window.showErrorMessage("Error starting Docker", { modal: true, detail: (e as Error).toString() }, "Install Docker Desktop", "Try again").then((value) => {
```

Comment:
This code looks to me like you're going to recommend installing Docker Desktop even in cases where the engine is just down.  You are able to resolve the difference between the docker cli available and the docker cli being present but the docker engine not running.

Reply:
>recommend installing Docker Desktop even in cases where the engine is just down

Yeah, that was the idea. We can make this modal more actionable with some changes.

Reply:
I just don't think we should recommend re-installing docker desktop when it's likely just that the docker engine is not running.  I think that's just wrong because it makes it seem like Docker Desktop is somehow broken but it's just  ... not running.

Reply:
Alright, this change is committed.